docker/ftest-standalone-ping/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://jboss.org/schema/arquillian"
+    xsi:schemaLocation="http://jboss.org/schema/arquillian
+    http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+    <extension qualifier="docker">
+        <property name="machineName">dev</property>
+        <property name="dockerContainers">
+            pingpong:
+              image: jonmorehouse/ping-pong
+              exposedPorts: [8080/tcp]
+              await:
```

Comment:
yes, it is a copy paste from `ftest-standalone`. I can push again fixing this, but basically I followed the standalone example.